src/snark/Makefile

```diff
@@ -44,80 +49,80 @@ ifneq ($(NO_SUPERCOP),1)
 endif
 
 LIB_SRCS = \
-	src/algebra/curves/alt_bn128/alt_bn128_g1.cpp \
-	src/algebra/curves/alt_bn128/alt_bn128_g2.cpp \
-	src/algebra/curves/alt_bn128/alt_bn128_init.cpp \
-	src/algebra/curves/alt_bn128/alt_bn128_pairing.cpp \
-	src/algebra/curves/alt_bn128/alt_bn128_pp.cpp \
-	src/common/profiling.cpp \
-	src/common/utils.cpp \
-	src/gadgetlib1/constraint_profiling.cpp \
+	libsnark/algebra/curves/alt_bn128/alt_bn128_g1.cpp \
```

Comment:
Is there any particular reason why libsnark was moved from `src` to `libsnark` directory?

Reply:
I checkout the latest commit b73b709,  there are src and libsnark in bitcoinz/src/snark/ directory, I think libsnark is the new version, am i right?  I also check github.com/zcash/zcash/src/snark, there is only libsnark sub-directory

Reply:
Interesting, I'll have to research that more. I'm not the maintainer of this project, so I don't know until I dig.